applications/ThermalDEMApplication/tests/auxiliary_functions.py

```diff
@@ -0,0 +1,135 @@
+import os
+import json
+import shutil
+import multiprocessing
+import KratosMultiphysics
+import KratosMultiphysics.ThermalDEMApplication as ThermalDEM
+import KratosMultiphysics.ThermalDEMApplication.thermal_dem_analysis
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+from KratosMultiphysics import Logger
+
+result_print = True
+
+#---------------------------------------------------------------------------------
+class controlledExecutionScope:
```

Comment:
See [here](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide), the rules for Class names